pkg/builtin/manifest.yml

```
@@ -78,6 +78,34 @@ extensions:
               type: string
               title: Cesium Ion API access token
               description: Cesium Ion account users may use their personal API keys to be able to use their Cesium Ion assets(tile data, 3D data, etc) with their project.
+        - id: cameraLimiter
+          title: Camera Limiter
+          description: Set the camera limiting box.
+          fields:
+            - id: cameraLimitterEnabled
+              type: bool
+              title: Enable
+              defaultValue: false
+              description: Enable camera limiter
+            - id: cameraLimitterShowHelper
+              type: bool
+              title: Show Helper
+              defaultValue: false
+              description: Display the limiter boundaries.
+            - id: cameraLimitterTargetArea
+              type: camera
+              title: Target max height  
```

```suggestion
              title: Center position
```

confusing field names:

- cameraLimitterTargetArea -> Max height
- cameraLimitterTargetWidth -> Width
- cameraLimitterTargetLength -> Length

My suggestion:

- cameraLimitterTargetArea -> Center position
- cameraLimitterTargetWidth -> Width
- cameraLimitterTargetHeight (renaming from Length) -> Height

---

@lavalse , please check, what do you think?

---

I think using Width and Height is ok. Let's use Width and Height.

and how about:
cameraLimitterTargetArea  -> Max zoom out.
because if we call this "Center position" yes it is center , but the most import for this camera is the how far away from the land, and at same time we didn't show the center as a icon there. I think the "Max zoom out" may be better.

---

Discussed with red and it's ok